knowledge_repo/app/static/css/pages/base.css

```diff
@@ -4,22 +4,29 @@
 }
 
 .navbar {
-    background-color: #565a5c;
-    border-color: #565a5c;
-    color: #fff;
-    box-shadow: 0px 1px 5px rgba(0,0,0,0.5);
+    background-color: #fff;
+    border-color: #d2d2d2;
+    color: #000;
+    box-shadow: 0px 1px 3px rgba(0, 0, 0, 0.15);
     z-index: 10000;
 }
 .navbar-inverse .navbar-nav>li>a {
-  color: #fff;
+  color: #000000;
 }
+
+.navbar-inverse .navbar-nav>li>a:focus, .navbar-inverse .navbar-nav>li>a:hover {
+  color: #000;
+}
+
 .navbar .navbar-brand {
   float: none;
-  color: #fff;
+  color: #000;
   display: flex;
   align-items: center;
   padding-left: 0px;
   padding-right: 0px;
+  font-weight: bolder;
```

Comment:
is `bolder` what you want here? That's saying this text should be bolder than the parent's element. It might be better to set this to `700` instead

Reply:
👍